backend/tournesol/views/stats.py

```diff
@@ -4,13 +4,17 @@
 from dataclasses import dataclass
 from typing import List
 
+import logging
```

Comment:
`logging` is imported but not used, let's not forget it : )

You can run `docker exec -i tournesol-dev-api ./scripts/ci/lint.sh` to easily spot little mistakes like this in the code.

Reply:
Thank you for the advices as well apart from the work, they are really helpful!